cogs/voice.py

```diff
@@ -1,12 +1,12 @@
-import discord
 from discord.ext import commands
 
-class Voice(commands.Cog):
-    def __init__(self, bot):
-        self.bot = bot
+from cogs.base import BaseCog
+
+
+class Voice(BaseCog):
 
     def _get_current_voice_client(self, ctx):
-        for vc in self.bot.voice_clients:
+        for vc in ctx.bot.voice_clients:
```

Comment:
Hmm, I guess this doesn't really matter since `since.bot` and `ctx.bot` should always point to the same thing

Reply:
Yes, this is just for consistency.